Check/Content/ContentEntityTypes.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * @file
+ * Contains \SiteAudit\Check\Content\ContentTypes.
+ */
+
+/**
+ * Class SiteAuditCheckContentContentEntityTypes.
+ */
+class SiteAuditCheckContentContentEntityTypes extends SiteAuditCheckAbstract {
+  /**
+   * Implements \SiteAudit\Check\Abstract\getLabel().
+   */
+  public function getLabel() {
+    return dt('Content entity types');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getDescription().
+   */
+  public function getDescription() {
+    return dt('Available content entity types and counts');
+  }
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultFail().
+   */
+  public function getResultFail() {}
+
+  /**
+   * Implements \SiteAudit\Check\Abstract\getResultInfo().
+   */
+  public function getResultInfo() {
+    $ret_val = '';
+
+    if (empty($this->registry['content_entity_type_counts'])) {
+      if (drush_get_option('detail')) {
+        return dt('No nodes exist.');
+      }
+      return $ret_val;
+    }
+    if (drush_get_option('html') == TRUE) {
+      $ret_val .= '<table class="table table-condensed">';
+      $ret_val .= '<thead><tr><th>' . dt('Content Entity') . '</th><th>' . dt('Bundle') . '</th><th>' . dt('Node Count') . '</th></tr></thead>';
+      foreach ($this->registry['content_entity_type_counts'] as $entity_type => $bundles) {
+        foreach ($bundles as $bundle => $count) {
+          $ret_val .= "<tr><td>$entity_type</td><td>$bundle</td><td>$count</td></tr>";
+        }
+      }
+      $ret_val .= '</table>';
+    }
+    else {
+      $rows = 0;
+      foreach ($this->registry['content_entity_type_counts'] as $entity_type => $bundles) {
+        if ($rows++ > 0) {
+          $ret_val .= PHP_EOL;
+          if (!drush_get_option('json')) {
+            $ret_val .= str_repeat(' ', 4);
+          }
+        }
+        $ret_val .= dt('Entity:') . $entity_type . dt(', Total Nodes: ') . $this->registry['node_count'][$entity_type];
```

Comment:
Simplify, use `dt` variable replacement, something similar to:

```
dt('Entity: @entity_type, Count: @count', array('@entity_type' => $entity_type', '@count' => $count));
```

Also, still using Node in both the label and the variable; please refactor to entity.